packages/atomic/src/components/insight/user-actions/atomic-insight-user-actions-timeline/atomic-insight-user-actions-timeline.tsx

```diff
@@ -0,0 +1,242 @@
+import {Component, h, State, Prop} from '@stencil/core';
+import {
+  buildInsightUserActions,
+  InsightUserActions,
+  InsightUserActionsState,
+  InsightUserSession,
+} from '../..';
+import ArrowDownIcon from '../../../../images/arrow-full-down.svg';
+import ArrowUpIcon from '../../../../images/arrow-up.svg';
+import {
+  InitializableComponent,
+  InitializeBindings,
+  BindStateToController,
+} from '../../../../utils/initialization-utils';
+import {Button} from '../../../common/button';
+import {NoItemsContainer} from '../../../common/no-items/container';
+import {MagnifyingGlass} from '../../../common/no-items/magnifying-glass';
+import {InsightBindings} from '../../atomic-insight-interface/atomic-insight-interface';
+
+/**
+ * This component displays all the actions performed by a user around the time they created a case.
+ * The actions are grouped into multiple sessions, including the session during which the case was created,
+ * the sessions preceding the case creation and the sessions following the case creation.
+ *
+ * @component
+ * @example
+ * <AtomicInsightUserActionsTimeline userId={'123'} caseCreationDate={'2024-08-15T10:00:00Z'} />
+ *
+ */
+@Component({
+  tag: 'atomic-insight-user-actions-timeline',
+  styleUrl: 'atomic-insight-user-actions-timeline.pcss',
+  shadow: true,
+})
+export class AtomicInsightUserActionsTimeline
+  implements InitializableComponent<InsightBindings>
+{
+  @InitializeBindings() public bindings!: InsightBindings;
+  public userActions!: InsightUserActions;
+
+  @BindStateToController('userActions')
+  @State()
+  public userActionsState!: InsightUserActionsState;
+  public error!: Error;
+
+  /**
+   * The ID of the user whose actions are being displayed.
+   */
+  @Prop() public userId!: string;
+  /**
+   * The date and time when the case was created.
+   */
+  @Prop() public ticketCreationDate!: string;
+
+  public initialize() {
+    this.userActions = buildInsightUserActions(this.bindings.engine, {
+      options: {ticketCreationDate: this.ticketCreationDate},
+    });
+
+    this.userActions.fetchUserActions(this.userId);
+  }
```

Comment:
There's no difference in atomic between the init and the `connectedCallback` for example?

Reply:
Actually there is, the initialize method is a custom method and not a method that comes by default with the stencil component. 
This method is called by the stencil decorator `BindStateToController` we are using in this component.